books/free-programming-books-langs.md

```diff
@@ -1230,6 +1230,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 #### Next.js
 
+* [Crash Course Next.js 2021](https://www.youtube.com/watch?v=mTz0GXj8NN0&ab_channel=TraversyMedia)
 * [Mastering Next.js](https://masteringnextjs.com)
```

Comment:
LGTM. It has chapter timestamps in timeline.

Please, apply next patch to use right title, remove URL track parameters and complete authoring:
```suggestion
* [Mastering Next.js](https://masteringnextjs.com)
* [Next.js Crash Course 2021](https://www.youtube.com/watch?v=mTz0GXj8NN0) - Brad Traversy (Traversy Media)
```